atom/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -8,13 +8,57 @@
 
 #import <Cocoa/Cocoa.h>
 
+#include "atom/browser/mac/atom_application.h"
 #include "atom/browser/mac/dict_util.h"
 #include "atom/common/native_mate_converters/gurl_converter.h"
 #include "atom/common/native_mate_converters/value_converter.h"
 #include "base/strings/sys_string_conversions.h"
 #include "base/values.h"
+#include "native_mate/object_template_builder.h"
 #include "net/base/mac/url_conversions.h"
 
+namespace mate {
+template <>
+struct Converter<NSAppearance*> {
+  static bool FromV8(v8::Isolate* isolate,
+                     v8::Local<v8::Value> val,
+                     NSAppearance** out) {
+    if (val->IsNull()) {
+      *out = nil;
+    } else {
+      std::string name;
+      if (!mate::ConvertFromV8(isolate, val, &name)) {
+        return false;
+      }
+      if (name == "light") {
+        *out = [NSAppearance appearanceNamed:NSAppearanceNameAqua];
+        return true;
+      } else if (name == "dark") {
+        *out = [NSAppearance appearanceNamed:NSAppearanceNameDarkAqua];
+        return true;
+      }
+
+      return false;
+    }
+    return true;
+  }
+
+  static v8::Local<v8::Value> ToV8(v8::Isolate* isolate, NSAppearance* val) {
+    if (val == nil) {
+      return v8::Null(isolate);
+    }
+
+    if (val.name == NSAppearanceNameAqua) {
+      return mate::ConvertToV8(isolate, "light");
+    } else if (val.name == NSAppearanceNameDarkAqua) {
```

Comment:
Inconsistent_ use of `else if` after a return clause.

This is a style thing that I would let slide but there's a conditional return just above this paragraph and another return immediately following, making the inconsistency more jarring.

This would be good:

```cc
if (val == nil) {
  return v8::Null(isolate);
}
if (val.name == NSAppearanceNameAqua) {
  return mate::ConvertToV8(isolate, "light");
}
if (val.name == NSAppearanceNameDarkAqua) {
  return mate::ConvertToV8(isolate, "dark");
}
return mate::ConvertToV8(isolate, "unknown")
```

This would also be OK:

```cc
if (val == nil) {
  return v8::Null(isolate);
} else if (val.name == NSAppearanceNameAqua) {
  return mate::ConvertToV8(isolate, "light");
} else if (val.name == NSAppearanceNameDarkAqua) {
  return mate::ConvertToV8(isolate, "dark");
} else {
  return mate::ConvertToV8(isolate, "unknown")
}
```